[BUG] Fix metadata tests.

After merging this PR https://github.com/aquarist-labs/ceph/pull/44 versioned objects metadata tests fail to build.

```bash
/srv/ceph/src/test/rgw/sfs/test_rgw_sfs_sqlite_versioned_objects.cc:86:12: error: 'struct rgw::sal::sfs::sqlite::DBOPObjectInfo' has no member named 'bucket_name'; did you mean 'bucket_id'?
   86 |     object.bucket_name = bucketname;
      |            ^~~~~~~~~~~
      |            bucket_id
ninja: build stopped: subcommand failed.
Error: Process completed with exit code 1.
```

--- a/src/test/rgw/sfs/test_rgw_sfs_sqlite_versioned_objects.cc
+++ b/src/test/rgw/sfs/test_rgw_sfs_sqlite_versioned_objects.cc
@@ -67,6 +67,7 @@ protected:
     SQLiteBuckets buckets(conn);
     DBOPBucketInfo bucket;
     bucket.binfo.bucket.name = bucketname;
+    bucket.binfo.bucket.bucket_id = bucketname;
     bucket.binfo.owner.id = username;
     buckets.store_bucket(bucket);
   }
@@ -83,7 +84,7 @@ protected:
 
     DBOPObjectInfo object;
     object.uuid.parse(object_id.c_str());
-    object.bucket_name = bucketname;
+    object.bucket_id = bucketname;
     object.name = "test_name";
     object.size = rand();
     object.etag = "test_etag";

